Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+                }
+            }
+            return null;
+        }
+
+        public static bool InstallApp(string appFullPath, ConnectInfo connectInfo, bool waitForDone = true)
+        {
+            try
+            {
+                // Calc the cert and dep paths
+                string fileName = Path.GetFileName(appFullPath);
+                string certFullPath = Path.ChangeExtension(appFullPath, ".cer");
+                string certName = Path.GetFileName(certFullPath);
+                string depPath = Path.GetDirectoryName(appFullPath) + @"\Dependencies\x86\";
+
+                // Post it using the REST API
+                WWWForm form = new WWWForm();
+
+                // APPX file
+                FileStream stream = new FileStream(appFullPath, FileMode.Open, FileAccess.Read, FileShare.Read);
+                BinaryReader reader = new BinaryReader(stream);
+                form.AddBinaryData(fileName, reader.ReadBytes((int)reader.BaseStream.Length), fileName);
+                stream.Close();
+
+                // CERT file
+                stream = new FileStream(certFullPath, FileMode.Open, FileAccess.Read, FileShare.Read);
+                reader = new BinaryReader(stream);
+                form.AddBinaryData(certName, reader.ReadBytes((int)reader.BaseStream.Length), certName);
+                stream.Close();
+
+                // Dependencies
+                FileInfo[] depFiles = (new DirectoryInfo(depPath)).GetFiles();
+                foreach (FileInfo dep in depFiles)
+                {
+                    stream = new FileStream(dep.FullName, FileMode.Open, FileAccess.Read, FileShare.Read);
+                    reader = new BinaryReader(stream);
+                    string depFilename = Path.GetFileName(dep.FullName);
+                    form.AddBinaryData(depFilename, reader.ReadBytes((int)reader.BaseStream.Length), depFilename);
+                    stream.Close();
+                }
+
+                // Credentials
+                Dictionary<string, string> headers = form.headers;
+                headers["Authorization"] = "Basic " + EncodeTo64(connectInfo.User + ":" + connectInfo.Password);
+
+                // Unity places an extra quote in the content-type boundary parameter that the device portal doesn't care for, remove it
+                if (headers.ContainsKey("Content-Type"))
+                {
+                    headers["Content-Type"] = headers["Content-Type"].Replace("\"", "");
+                }
+
+                // Query
+                string query = string.Format(kAPI_InstallQuery, connectInfo.IP);
+                query += "?package=" + WWW.EscapeURL(fileName);
+                WWW www = new WWW(query, form.data, headers);
+                DateTime queryStartTime = DateTime.Now;
+                while (!www.isDone &&
+                       ((DateTime.Now - queryStartTime).TotalSeconds < kTimeOut))
+                {
+                    System.Threading.Thread.Sleep(10);
```

Comment:
You actually can use coroutines you just need to manually tick them in the update function of your editor script:
    public void Update()
    {
	   // Coroutine is finished if MoveNext() == false;
	   if (!loadingRoutine.MoveNext())
       {
	      m_bLoaded = true;
          PostLoadFunction();
       }
    }